src/pages/index.js

```diff
@@ -218,23 +223,30 @@ function handleAvatarSubmit(evt) {
 }
 
 function handlePostCard(evt) {
-  evt.preventDefault();
-  const postCardSubmitBtn = evt.submitter;
-  setButtonText(postCardSubmitBtn, true);
-
+  renderLoading(true, cardAddSubmitButton);
   api
-    .postCard({ name: newPostCaption.value, link: newPostImageLink.value })
-    .then((data) => {
-      const newCard = getCardElement(data);
-
-      cardsList.prepend(newCard);
-
-      closeModal(addCardModal);
-      evt.target.reset();
+    .postCard(data)
+    .then((cardData) => {
+      cardSection.addItem(createCard(cardData));
+      cardAddForm.reset();
+      cardAddModal.close();
```

Comment:
Please, check if this code is correct. You shouldn't have changed it from the last time